scheduler/views.py

```diff
@@ -38,17 +47,38 @@ def upload(request: HttpRequest) -> HttpResponse:
     return render(request, "upload.html")
 
 
-def show_calendar(request: HttpRequest) -> HttpResponse:
-    """ to do """
-    times = pd.date_range('2019-12-02T08:00:00.000Z', '2019-12-02T22:00:00.000Z', freq='15T')
-    rooms = Auditorium.objects.all()
+def show_rooms_schedule(request: HttpRequest) -> HttpResponse:
+    """Render the auditorium schedule page"""
+    if_chosen = False
```

Comment:
To co wyżej. Ta flaga mogłaby natomiast pilnować czy dla danej grupy/sali/prowadzącego istnieją jakieś zajęcia. W tej chwili jeśli zajęć nie ma (bo ktoś je np. odwołał i usunął) to znika jedynie kalendarz, tekst powyżej i pole na detale zostaje co wygląda trochę średnio